internal/statemachine/helper_test.go

```diff
@@ -1265,3 +1265,19 @@ func TestCheckCustomizationSteps(t *testing.T) {
 		})
 	}
 }
+
+// TestFailedMountNewFS tests failures in the mountTempFS function
+func TestFailedMountNewFS(t *testing.T) {
+	t.Run("test_failed_mount_new_fs", func(t *testing.T) {
```

Comment:
nitpick: it got renamed to "mountTempFS" and that should also be reflected in the tests.

Reply:
ah yeah, case sensitive sed missed this one. Thanks!

Reply:
`test_failed_mount_new_fs` could be adopted, too... but meh.